src/learning/infer.py

```diff
@@ -0,0 +1,31 @@
+import torch
+import torchvision
+from torchvision.utils import save_image
+import utils
```

Comment:
only import get_transform, as it is the only thing you are using from utils?

Reply:
Done.